e2e/pages/events/manageOrders.page.js

```diff
@@ -75,6 +78,11 @@ const selectChildren = async (option, indexes = []) => {
   await I.runAccessibilityTest();
 };
 
+const enterDirections = async (text) => {
```

Comment:
how about having this method also populate the title as well? Then we can make sure we are asserting both fields are present

Reply:
Updated to check both fields are present. I opted for a new function for entering the title because I think that will look a bit clearer